src/components/Charts/ChartBlock.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react';
+import findIndex from 'lodash/findIndex';
+import isNull from 'lodash/isNull';
+import { Projections } from 'common/models/Projections';
+import { Region } from 'common/regions';
+import { SectionHeader } from 'components/SharedComponents';
+import ChartTabs from 'components/NewLocationPage/ChartTabs/ChartTabs';
+import {
+  Tab,
+  InactiveTabWrapper,
+} from 'components/NewLocationPage/ChartTabs/ChartTab.style';
+import {
+  ChartGroup,
+  MetricChartInfo,
+  getValueInfo,
+} from 'components/Charts/Groupings';
+import { MetricValues } from 'common/models/Projections';
+import ChartFooter from 'components/NewLocationPage/ChartFooter/ChartFooter';
+import { Metric } from 'common/metricEnum';
+
+const ChartBlock: React.FC<{
+  isMobile: boolean;
+  region: Region;
+  stats: MetricValues;
+  projections: Projections;
+  group: ChartGroup;
+  clickedStatMetric: Metric | null;
+  groupRef: React.RefObject<HTMLDivElement>;
+}> = ({ projections, stats, group, region, groupRef, clickedStatMetric }) => {
+  const { metricList, groupHeader } = group;
+
+  // TODO (chelsi) - revisit placement of these state/setState variables
+  const [activeTabIndex, setActiveTabIndex] = useState(0);
+  const onChangeTab = (newTabIndex: number) => {
+    setActiveTabIndex(newTabIndex);
+  };
+
+  // Selects correct tab after clicking summary stat + scrolling to chart block
```

Comment:
It feels like we might want to move the state up, however, it will be a bit weird to control all the tabs from above. We can leave it for now unless we come up with a solution!

Reply:
yeah, [added a todo for this](https://github.com/covid-projections/covid-projections/pull/3817/files#diff-be721b17c599144023d04f06fdf031526e3e773d196005821689ac6f87f92131R32)